python/cuml/test/test_fil.py

```diff
@@ -91,16 +96,15 @@ def _build_and_save_xgboost(model_path,
                                         unit_param(5),
                                         quality_param(50),
                                         stress_param(90)])
+@pytest.mark.parametrize('n_classes', [2, 5])
```

Comment:
We should probably have a test with a lot of classes as well... what's the practical limit?

Reply:
added 25 class test. 5000 class test is present in C++ tests; and training xgboost on 0.5M rows with this many classes took too long. Skipping 5K class test in python for now.